lnbits/core/views/generic.py

```diff
@@ -111,26 +102,10 @@ async def extensions_install(
         installed_exts_ids = []
 
     try:
-        ext_id = activate or deactivate
-        all_extensions = get_valid_extensions()
-        ext = next((e for e in all_extensions if e.code == ext_id), None)
-        if ext_id and user.admin:
-            if deactivate:
-                settings.lnbits_deactivated_extensions.add(deactivate)
-            elif activate:
-                # if extension never loaded (was deactivated on server startup)
-                if ext_id not in sys.modules.keys():
-                    # run extension start-up routine
-                    core_app_extra.register_new_ext_routes(ext)
-
-                settings.lnbits_deactivated_extensions.remove(activate)
-
-            await update_installed_extension_state(
-                ext_id=ext_id, active=activate is not None
-            )
-
-        all_ext_ids = [ext.code for ext in all_extensions]
-        inactive_extensions = await get_inactive_extensions()
```

Comment:
this logic is moved to: `/api/v1/extension/{ext_id}/activate` and `/api/v1/extension/{ext_id}/deactivate`